Exercise-15/assets/css/styles.css

```diff
@@ -0,0 +1,286 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+}
+
+img {
+  width: 100%;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+}
+
+.justify-right {
+  justify-content: end;
+}
+
+.container {
+  width: 67.5%;
+  margin: 0 auto;
+}
+
+.width50 {
+  width: 50%;
+}
+
+/* header section */
+
+.header-content {
+  box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  top: 0;
+  background-color: #ffffff;
+  z-index: 1;
+}
+
+.logo {
+  font-size: 20px;
+  font-weight: 600;
+  margin: 18px 0;
+}
+
+.profile-name {
+  font-size: 11px;
+  margin: 0 10px;
+}
+
+.profile-image {
+  height: 25px;
+  width: 25px;
+  border-radius: 50%;
+}
+
+/* main section */
+
+main {
+  background-color: #f5f5f5;
+  width: 100%;
+  padding: 22px 0;
+}
+
+main div.container {
```

Comment:
could this selector be just .container

Reply:
No, i have common style in container and apart from that i need these styles specific for this container inside main